.github/workflows/unit-tests-integrations.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         node-version: [18.x, 20.x]
-        package: [anthropic, azure-openai, cloudflare, cohere, community, exa, google-common, google-gauth, google-genai, google-vertexai, google-vertexai-web, google-webauth, groq, mistralai, mongo, nomic, openai, pinecone, qdrant, redis, textsplitters, weaviate, yandex]
+        package: [anthropic, azure-openai, cloudflare, cohere, community, exa, google-common, google-gauth, google-genai, google-vertexai, google-vertexai-web, google-webauth, groq, mistralai, mongo, nomic, openai, pinecone, qdrant, redis, textsplitters, weaviate, yandex, qianfan]
```

Comment:
Can we call the new package `@langchain/baidu` or `@langchain/baidu-qianfan`?

Reply:
OK, I will change the name to  @langchain/baidu-qianfan